src/Umbraco.Web.BackOffice/Security/BackOfficeAntiforgery.cs

```diff
@@ -33,14 +33,12 @@ public BackOfficeAntiforgery(IOptionsMonitor<GlobalSettings> globalSettings)
         {
             x.HeaderName = Constants.Web.AngularHeadername;
             x.Cookie.Name = Constants.Web.CsrfValidationCookieName;
+            x.Cookie.SecurePolicy = globalSettings.CurrentValue.UseHttps ? CookieSecurePolicy.Always : CookieSecurePolicy.SameAsRequest;
```

Comment:
Why is this added? This doesn't seem to be relevant for this PR...

Reply:
It's not. That's just to resolve a merge conflict the umbraco team encountered.

As for the changes I made
- Umbraco.Web.Backoffice didn't depend have an existing dependency on Umbraco.Infrastructure, so I added the Markdown library directly to it to avoid messing with dependencies. Probably not a difficult change, but I'm not sure whether there is a reason that dependency didn't already exist.
- It's difficult to manually handle markdown edge cases with regex, which is why I went with the markdown library. Also, any issues parsing the markdown text would be an issue with markdown library and would also impact the built in "Markdown Editor" data type. In my case, I needed to display "**" in the description, which is tricky to handle because it needed to check that there isn't a second "**" after it. Gets a bit messier if there are multiple sets.
- You're right that removing the simpleMarkdown filter is technically a breaking change, though it's not a documented filter and so the risk of its removal breaking something is very low.
- Broken markup was also possible before this change, so making sure that doesn't happen is up to the developer.
- Notifications are certainly another way this could have been handled. Was still learning umbraco 9 at the time I created this PR.

Reply:
If removing the `simpleMarkdown` filter is a breaking change, then we can just re-instate the files even though we're not using them. Other than that, existing markdown in descriptions will just be parsed by the markdown library and shouldn't change.

For the rest I concur with @jaandrews. 

For adding a dependency, maybe it needs to be a call to a library with that dependency, but I also don't know if a dependency on `Umbraco.Infrastructure` from `Umbraco.Web.Backoffice` would be welcomed.